wikicurses/settings.py

```diff
@@ -1,16 +1,29 @@
+import sys
 import os
 import json
 import collections
 import configparser
+import shutil
 from wikicurses import formats
 from urllib.parse import urlparse
 
-default_configdir = os.environ['HOME'] + '/.config'
+# TODO: This may cause issues with python 2.7; however, as of 01OCT2019, it is nearing sunset
```

Comment:
Running Wikicurses in Python 2 already fails immediately (syntax errors from `yield from`, which was added later). So that shouldn't be a problem.

(I personally don't see any reason now to support Python 2 in an application; maybe in a library.)